server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/JmapRfc8621Configuration.scala

```diff
@@ -22,24 +22,28 @@ package org.apache.james.jmap.model
 import java.net.URL
 
 import org.apache.commons.configuration2.Configuration
-import org.apache.james.jmap.routes.UploadRoutes
-import org.apache.james.jmap.routes.UploadRoutes.Size
 import org.apache.james.util
 import org.apache.james.util.Size
 
+import scala.util.{Failure, Success, Try}
+
 
 object JmapRfc8621Configuration {
   val LOCALHOST_URL_PREFIX: String = "http://localhost"
   private def from(urlPrefixString: String): JmapRfc8621Configuration = JmapRfc8621Configuration(urlPrefixString)
   var LOCALHOST_CONFIGURATION: JmapRfc8621Configuration = from(LOCALHOST_URL_PREFIX)
   val URL_PREFIX_PROPERTIES: String = "url.prefix"
 
-  val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.mb.size"
-  val UPLOAD_LIMIT_30_MB: String = "30M"
+  val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.size"
+  val UPLOAD_LIMIT_30_MB: Long = 30 * 1024 * 1024
   var MAXSIZE_UPLOAD: Option[util.Size] = None
```

Comment:
Why do we have some var inside an object? Can we change this?

Reply:
You are mixing things. MAXSIZE_UPLOAD needs to be a field of the case class.

Reply:
We need something to hold the value and less breaking change, no?

Reply:
No

1. We should **never** use var

2. Move this to the case class, you are hardcoding a singleton pattern.